polaris.shopify.com/src/data/navItems.ts

```diff
@@ -193,6 +193,13 @@ export const foundationsNavItems: NavItemWithIcon[] = [
   {
     title: "Patterns",
     children: [
+      {
+        title: "Design Patterns",
+        url: "/foundations/patterns/design-patterns",
+        icon: FoundationsIcons.LoadingIcon,
```

Comment:
We need to add the right icon to this page before it gets merged in
<img width="456" alt="Screen Shot 2022-08-04 at 7 15 23 PM" src="https://user-images.githubusercontent.com/4642404/182976004-218a07c0-b98f-42d0-9c6f-b6780b99696c.png">